fip-6.md

```diff
@@ -0,0 +1,141 @@
+---
+fip: 6
+title: Provide ability to deactivate FIO Addresses and Domains
+status: Draft
+type: Functionality
+author: Casey Gardiner <casey@dapix.io>
+created: 2020-04-22
+updated: 
+---
+
+## Abstract
+This FIP implements the following:
+* Adds the ability for FIO Address owners the ability to deactivate stale or unused addresses.
+* Adds the ability for FIO Domain owners the ability to set their domain as expired.
+* Adds new API end points for FIO Domain deactivation and FIO Address burn. 
+* Adds new fees for FIO Domain deactivation and FIO Address burn.
+
+## Motivation
+Presently the FIO Blockchain only burns and removes old addresses and domains once they reach their expiration date. Added functionality to remove these addresses and domains at any given time gives the owners flexibility.
+
+## Specification
+### Burn Addresses
+#### New end point: *deactivate_fio_address* 
```

Comment:
Burn or deactivate?

Reply:
Addresses are not being burned. Just slotted to be burned when burnexpired is called.